app/src/index.css

```diff
@@ -19,6 +19,10 @@
     scroll-behavior: smooth;
   }
 
+  .bn-notify-custom .bn-notify-notification {
+    font-family: Source Code Pro, monospace;
```

Comment:
We do. I believe the notification library is actually it's own Svelte app, so our styles don't apply by default. Before adding this line, the font wasn't updated, so that's how I really know it's needed 😛

A bit more info here if you're interested: https://docs.blocknative.com/notify#customizing-css

Reply:
Ahhh, that makes sense -- thanks for the context!!